core/frontend/src/components/vehiclesetup/configuration/failsafes/Failsafes.vue

```diff
@@ -0,0 +1,200 @@
+          image: (await import('@/assets/img/configuration/failsafes/heartbeat.svg')).default as string,
+
+          params: [
+            {
+              replacementTitle: 'Timeout',
+              icon: 'mdi-timer',
+              name: 'FS_GCS_TIMEOUT',
+              optional: true,
+            },
+            {
+              replacementTitle: 'Action',
+              icon: 'mdi-lightning-bolt',
+              name: 'FS_GCS_ENABLE',
+              replacementDescription: '',
+            },
+          ],
+        },
+        {
+          name: 'Pilot Input Loss',
+          generalDescription: 'The pilot input failsafe is triggered when the vehicle does not receive '
+          + 'any pilot input for a given amount of time.',
+          image: (await import('@/assets/img/configuration/failsafes/pilot-input.svg')).default as string,
+          params: [
+            {
+              replacementTitle: 'Action',
+              icon: 'mdi-lightning-bolt',
+              name: 'FS_PILOT_INPUT',
+            },
+            {
+              replacementTitle: 'Timeout',
+              icon: 'mdi-timer-outline',
+              name: 'FS_PILOT_TIMEOUT',
+            },
+          ],
+        },
+        {
+          name: 'Leak detection',
+          generalDescription: 'Triggered when a leak is detected. We recoomend keeping control electronics dry.',
+          image: (await import('@/assets/img/configuration/failsafes/leak.svg')).default as string,
+          params: [
+            {
+              replacementTitle: 'Action',
+              icon: 'mdi-lightning-bolt',
+              name: 'FS_LEAK_ENABLE',
+            },
+            {
+              replacementTitle: 'Leak probe 1 type',
+              name: 'LEAK1_TYPE',
+            },
+            {
+              replacementTitle: 'Leak 1 pin',
+              name: 'LEAK1_PIN',
+            },
+            {
+              replacementTitle: 'Leak 1 logic level when dry',
+              name: 'LEAK1_LOGIC',
+            },
+          ],
+        },
+        {
+          name: 'Internal Pressure',
+          generalDescription: 'Triggered when the internal pressure is too high. this could help in detecting a leak'
+          + ' and in avoiding rapid unplanned disassembly.',
+          image: (await import('@/assets/img/configuration/failsafes/pressure.svg')).default as string,
+          params: [
+
+            {
+              replacementTitle: 'Maximum internal pressure',
+              name: 'FS_PRESS_MAX',
+            },
+            {
+              replacementTitle: 'Action',
+              icon: 'mdi-lightning-bolt',
+              name: 'FS_PRESS_ENABLE',
+            },
+          ],
+        },
+        {
+          name: 'Battery Level',
+          generalDescription: 'Triggered when the battery reaches given thresholds.\n This can help in avoiding '
+          + 'damage to the battery and potentially loss of the vehicle.',
```

Comment:
```suggestion
          name: 'Low Battery',
          generalDescription: 'Triggers when the voltage goes below specified thresholds.\n This can help to avoid '
          + 'damage to the battery and potentially loss of the vehicle.',
```